src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -0,0 +1,345 @@
+from copy import deepcopy
+from typing import Literal, Optional
+
+import numpy as np
+from pydantic import FilePath, validate_call
+from pynwb import NWBFile
+from pynwb.image import ImageSeries
+
+from .video_utils import VideoCaptureContext
+from ....basedatainterface import BaseDataInterface
+from ....tools import get_package
+from ....tools.nwb_helpers import get_module
+from ....utils import dict_deep_update, get_base_schema, get_schema_from_hdmf_class
+
+
+class ExternalVideoInterface(BaseDataInterface):
+    """Data interface for writing videos as external_file ImageSeries."""
+
+    display_name = "Video"
+    keywords = ("video", "behavior")
+    associated_suffixes = (".mp4", ".avi", ".wmv", ".mov", ".flx", ".mkv")
+    # Other suffixes, while they can be opened by OpenCV, are not supported by DANDI so should probably not list here
+    info = "Interface for handling standard video file formats."
+
+    @validate_call
+    def __init__(
+        self,
+        file_paths: list[FilePath],
+        verbose: bool = False,
+        *,
+        video_name: str = "ExternalVideo",
+    ):
+        """
+        Initialize the interface.
+
+        This interface handles multiple video segments and writes them as an ImageSeries with a link to external_file.
+        For writing videos internally with just an ImageSeries object, use the InternalVideoInterface.
+
+        Parameters
+        ----------
+        file_paths : list of FilePaths
+            Many video storage formats segment a sequence of videos over the course of the experiment.
+            Pass the file paths for this videos as a list in sorted, consecutive order.
+        verbose : bool, optional
+            If True, display verbose output. Defaults to False.
+        video_name : str, optional
```

Comment:
It always confuses me that optional in the typing system means that something can be None but in numpy style docstrings it means that it is not necessary to specifiy.

There is also default instead of optional which I think is more explicit. 

I know that Ben has a preference but I always forget which one is it.

https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
I feel like optional is fine, and it's everywhere in our codebase.

Reply:
> There is also default instead of optional which I think is more explicit.

This is the kind of change that I think has such low marginal value it's really not worth the time and effort discussing/implementing it.

Reply:
Agree, it is a preference of Ben though so was just letting you know.